app/views/content/subjects/maths/_article.html.erb

```diff
@@ -1,189 +1,133 @@
-    ) %>
-  </section>
-
   <section class="col col-720 col-horizontal col-space-m">
     <div>
-      <h2 class="heading-s">What you'll be teaching</h2>
-
-      <p>
-        In England, there are important aspects of maths to teach each age
-        group. As a secondary school teacher, maths education means teaching
-        pupils:
-      </p>
-    </div>
-
-    <div>
+      <h2>What you'll be teaching</h2>
+      <p>You'll teach the national curriculum. The themes you're likely to cover include: </p>
+      <p>11 to 16 year olds (key stage 3 and 4):</p>
       <ul class="vertical-tags">
         <li class="tag">Numbers</li>
         <li class="tag">Algebra</li>
-        <li class="tag">Ratio, portion and rates of change</li>
+        <li class="tag">Ratio, proportion and rates of change</li>
         <li class="tag">Geometry and measures</li>
-        <li class="tag">Probability</li>
-        <li class="tag">Statistics</li>
+        <li class="tag">Probability and statistics</li>
       </ul>
+
+      <p>16 to 18 year olds (key stage 5) - A or AS level core maths:</p>
+      <p>Compulsory content</p>
+      <ul class="vertical-tags">
+        <li class="tag">Analysis of data</li>
+        <li class="tag">Maths for personal finance</li>
+        <li class="tag">Estimation</li>
+        <li class="tag">Critical analysis of given data and models</li>
+      </ul>
+      <p>Optional content</p>
+      <ul class="vertical-tags">
+        <li class="tag">The normal distribution</li>
+        <li class="tag">Probabilities and estimation</li>
+        <li class="tag">Correlation and regression</li>
+        <li class="tag">Critical path and risk analysis</li>
+        <li class="tag">Expectation</li>
+        <li class="tag">Cost benefit analysis</li>
+        <li class="tag">Graphical methods</li>
+        <li class="tag">Rates of change</li>
+      </ul>
+
+    <p><a href="https://www.gov.uk/government/publications/national-curriculum-in-england-mathematics-programmes-of-study/national-curriculum-in-england-mathematics-programmes-of-study#key-stage-3">
+        Read the full national curriculum for more information about teaching maths in secondary schools</a>.</p>
+     <p>You can also search for past exam papers online to see the type of questions that were asked in previous years.</p>
+    <p>You may also get the opportunity to teach other subjects, particularly for 11 to 16 year olds and in some cases for post-16. This will vary depending on individual schools.
     </div>
+</section>
+
+<section class="col col-845 inset">
+    <%= render Content::PhotoQuoteListComponent.new(
+      [
+        {
+          image: "static/content/subjects/rotna-roy.jpeg",
+          heading: "Empowering my students",
+          text: "I enjoy showing my students that maths is essential in everyday life, whether it is identifying patterns, interpreting data, managing finances, or understanding the spaces and structures around us, everything uses maths.",
```

Comment:
```suggestion
          text: "I enjoy showing my students that maths is essential in everyday life. Whether it is identifying patterns, interpreting data, managing finances, or understanding the spaces and structures around us, everything uses maths.",
```
sentence feels very long - worth breaking it up?

Reply:
Still feel this sentence is a bit long with too many parts and commas to be easily readable!

Not sure if you disagree @philspray as this comment is still open :-)

Reply:
Sorry I must have missed this one for some reason. I agree it would help to break it up.